pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -685,10 +686,15 @@ async def run_repomdrecord_download(name, location_href, downloader):
                             with open(result.path, "r") as f:
                                 extra_files = json.loads(f.read())
                                 for data in extra_files["data"]:
+                                    filtered_checksums = {
+                                        digest: value
+                                        for digest, value in data["checksums"].items()
+                                        if digest in settings.ALLOWED_CONTENT_CHECKSUMS
```

Comment:
We have been bitten by "SHA == SHA1" - I don't have an example to hand, but I know there are several/many on how to notice SHA and convert it correctly. Do we need to do that here?

Reply:
Here's an example of what I was thinking about : https://github.com/pulp/pulp_rpm/blob/master/pulp_rpm/app/tasks/synchronizing.py#L631

Reply:
I do not think distribution tree metadata suffered from that, it seems to me to be a purely createrepo_c hiccup.